integration/ksm/ksm_test.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+set -e
+
+dir_path=$(dirname "$0")
+source "${dir_path}/../../lib/common.bash"
+source "${dir_path}/../../metrics/lib/common.bash"
+
+TEST_NAME="Run containers with KSM"
+KATA_KSM_THROTTLER="${KATA_KSM_THROTTLER:-yes}"
+RUNTIME="${RUNTIME:-kata-runtime}"
+PAYLOAD_ARGS="${PAYLOAD_ARGS:-tail -f /dev/null}"
+IMAGE="${IMAGE:-busybox}"
+WAIT_TIME="60"
+
+function setup() {
+	clean_env
+
+	check_processes
+
+	save_ksm_settings
+
+	set_ksm_aggressive
+
+	check_for_ksm
```

Comment:
I don't think you need to call this - if you read the code, what it does is append ` ksm` to the end of the `TEST_NAME` if KSM is enabled (to differentiate the results files generated, if you run the same test with and without KSM enabled). You already have `KSM` on the end of your `TEST_NAME`, so right now it will end up being called `KSM ksm` :-)